### DISCUSSION TITLE
Waiting for verification + token emails: how long until one gets the API token?

### DISCUSSION THREAD

**Discussion by AnneEstoppey** [DeepGenX/CodeGenX#35] [General]
Hello

I have sent a request for an API token this afternoon at about 1PM (Sunday 13th November). It's soon 4PM, and still no verification email in my inbox. Yes I have checked several times my spam folder as well. Yes I sent the request again in case I made a typo. Still nothing. 

Is there a queue for requesting the API token?
If yes, how much do we have to wait, approximately?

Thanks a lot in advance for information,
Cheers 😊

**Comment by yali93** [comment]
any update as I am in your same situation
